README.md

```diff
@@ -2396,6 +2396,11 @@ apache::vhost { "$::fqdn":
 
 Sets the [`ForceType`][] directive, which forces Apache to serve all matching files with a [MIME `content-type`][] matching this parameter's value.
 
+#### `add_charset`
+
+Lets Apache set custom content character sets per directory and/or file extension
+
+
```

Comment:
could you please remove this last empty line and move this param up into alphabetical order for the vhost type?